### PR TITLE
Adjust for mouse alleles

### PR DESCRIPTION
**Pull request by skrakau**
Mouse allele names do not contain "*", which is used to access the allele results. I added "H-2-" to this. 

Many thanks for contributing to nf-core/epitopeprediction!

Please fill in the appropriate checklist below (delete whatever is not relevant). These are the most common things requested on pull requests (PRs).

## PR checklist
 - [x] This comment contains a description of changes (with reason)
 - [ ] If you've fixed a bug or added code that should be tested, add tests!
 - [ ] If necessary, also make a PR on the [nf-core/epitopeprediction branch on the nf-core/test-datasets repo]( https://github.com/nf-core/test-datasets/pull/new/nf-core/epitopeprediction)
 - [x] Ensure the test suite passes (`nextflow run . -profile test,docker`).
 - [ ] Make sure your code lints (`nf-core lint .`).
 - [ ] Documentation in `docs` is updated
 - [ ] `CHANGELOG.md` is updated
 - [ ] `README.md` is updated

**Learn more about contributing:** https://github.com/nf-core/epitopeprediction/tree/master/.github/CONTRIBUTING.md

### CODE REVIEW

**Comment by skrakau** [comment]
Does currently not work when running pipeline on variants, due to missing reference. Make sure only run in combination with peptides (or later proteins).

**Comment by christopher-mohr** [comment]
Blocked by changes necessary in FRED2 for now.

**Comment by skrakau** [comment]
If a mouse allele is provided and `--input` is specified, it will complain and exit. However, it is a bit suboptimal, we should check in general supported models and alleles.

**Comment by skrakau** [comment]
> However, it is a bit suboptimal, we should check in general supported models and alleles.

this will be addressed in #44.

However, we should add a test for mouse alleles.

**Comment by skrakau** [comment]
:D so all tests run through even though the (only) mouse allele in the test is not supported yet in the current Fred2 version 2.0.6, the results will be empty.

 **This should still wait for a new FRED release**

(with https://github.com/FRED-2/Fred2/pull/242 and/or https://github.com/nf-core/epitopeprediction/pull/44 this would not happen again)

**Comment by christopher-mohr** [comment]
New FRED2 release finally there: https://github.com/FRED-2/Fred2/releases/tag/v2.0.7

File: bin/epaa.py [hunk 1]
```diff
@@ -808,7 +808,7 @@ def make_predictions_from_variants(variants_all, methods, alleles, minlength, ma
         df.reset_index(inplace=True)
 
         for c in df.columns:
-            if '*' in str(c):
+            if ('HLA-' in str(c)) or ('H-2-' in str(c)):
```
Comment (christopher-mohr) [inline bin/epaa.py:811]: Can we be sure that the allele name in the column contains the `HLA-` prefix? I am not sure right now.

Comment (christopher-mohr) [reply bin/epaa.py:811]: This is how it is currently done in `FRED2`: 

```
if name.startswith("H2-"):
     return MouseAllele(name, prob=prob)
if name.count("*") > 1:
     return CombinedAllele(name, prob=prob)
```

Comment (skrakau) [reply bin/epaa.py:811]: Hi @christopher-mohr ,
I think we can be sure. The `result` returned by `predict()` contains the allele as type `Allele`. (In `ANN.py` returned from `revert_allele_repr()` and in `External.py`, `PSSM.py`, `SVM.py` it is checked that the allele is an instance of type `Allele`.) When converting this to a `str` (in `epaa.py`) it should `return '%s-%s*%s:%s' % (str(self.organism), str(self.locus), str(self.supertype), str(self.subtype))` and thus contain `HLA-`.

**Review by christopher-mohr — APPROVED** [review]

**Pull request merged**